src/main/java/xyz/nucleoid/plasmid/storage/FriendSystem/FriendList.java

```diff
@@ -0,0 +1,60 @@
+package xyz.nucleoid.plasmid.storage.FriendSystem;
+
+import java.util.ArrayList;
+import java.util.UUID;
+/*
+    Please only touch this if you know what you are doing
+ */
+
+public class FriendList {
+    private int playerId;
+    private final ArrayList<UUID> friends = new ArrayList<UUID>(); // holds accepted friends
```

Comment:
These can be implemented with a `Set<UUID>`, which optimise the contains checks and also protects against duplicates :)

Reply:
Same for requests.